www/index.html

```diff
@@ -4,6 +4,11 @@
 <head>
   <meta charset="UTF-8">
   <title>Dose Amigos</title>
+
+  <!-- Auth0 Lock script -->
+  <script src="https://cdn.auth0.com/js/lock-9.0.min.js"></script>
```

Comment:
I'm about to test on the android to see if when running on a physical device if this is a problem.

I probably need to have it copied during the gulp build.